src/3d/qgsvectorlayerchunkloader_p.cpp

```diff
@@ -132,6 +134,7 @@ Qt3DCore::QEntity *QgsVectorLayerChunkLoader::createEntity( Qt3DCore::QEntity *p
   }
 
   Qt3DCore::QEntity *entity = new Qt3DCore::QEntity( parent );
+  entity->setObjectName( mFactory->mLayer->name() + "_" + mNode->tileId().text() );
```

Comment:
I don't think it's safe to access mLayer here -- I would store the name as a member string in the QgsVectorLayerChunkLoader constructor and use that here.

Reply:
I need to check that

Reply:
I made the fix but I do not understand the issue when using `mFactory->mLayer`! Can you explain me?

Reply:
Create entity is called after the job is finished, and there's a chance that the layer has been deleted in the meantime...